oauth2client/contrib/gce.py

```diff
@@ -84,7 +84,8 @@ def _refresh(self, http_request):
         """
         query = '?scope=%s' % urllib.parse.quote(self.scope, '')
         uri = META.replace('{?scope}', query)
-        response, content = http_request(uri)
+        headers = { 'Metadata-Flavor' : 'Google' }
```

Comment:
I'm all about moving to the latest version, so this is more a curiosity: Was there some kind of announcement about deprecating the old URI or an encouragement for users to upgrade?

Are there implications for the [other place][1] this library talks to the metadata API?

[1]: https://github.com/google/oauth2client/blob/8f8df1f5977ec76d50cb3dca9a31044cfd1fb9b4/oauth2client/client.py#L116